src/bag3_digital/layout/stdcells/memory.py

```diff
@@ -591,6 +591,7 @@ def draw_layout(self) -> None:
             seg = seg_dict['nor']
             seg_t1 = seg_dict['keep']
             seg_t0 = seg_dict['in']
+        print(f'nor:{seg}, keep: {seg_t1}, in: {seg_t0}')
```

Comment:
either remove the print statement or provide more context about the generator class name